src/main/scala/com/microsoft/hyperspace/util/PathUtils.scala

```diff
@@ -26,4 +26,12 @@ object PathUtils {
     val fs = path.getFileSystem(new Configuration)
     fs.makeQualified(path)
   }
+
+  /* Definition taken from org.apache.spark.sql.execution.datasources.PartitionAwareFileIndex. */
```

Comment:
Can you also take the comments for the function?

Reply:
thanks @imback82 , done